bccsp/pkcs11/pkcs11.go

```diff
@@ -215,9 +215,22 @@ func (csp *impl) generateECKey(curve asn1.ObjectIdentifier, ephemeral bool) (ski
 	session := csp.getSession()
 	defer csp.returnSession(session)
 
-	id := nextIDCtr()
-	publabel := fmt.Sprintf("BCPUB%s", id.Text(16))
-	prvlabel := fmt.Sprintf("BCPRV%s", id.Text(16))
+	keylabel := ""
+	updateSKI := false
+	if csp.altId == "" {
+		// Generate using the SKI process and then make keypair immutable according to csp.immutable
+		keylabel = fmt.Sprintf("BCP%s", nextIDCtr().Text(16))
```

Comment:
Why the change from BCPUB/BCPRV to BCP as part of this PR?
What does BCP stand for?

Reply:
Both BCPUB and BCPRV are temporary placeholders for the CKA_ID and CKA_LABEL.  Current code sets the value to the private key and public key to these values.

Keeping these two fields for that purpose made the modification more complex.  Given that the public key and private key should always have the same ID and Label, I felt it was a small change to make it more readable.  BCP stands for Blockchain Crypto Provider.

In any case, it does not impact the functionality, I can revert that change if there is a preference for that.